reactrestaurant/src/components/navtabs.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import Badge from './badge';
+
+function NavTabs() {
+    return (
+        <>
+            <ul className="nav nav-pills mb-3 justify-content-center" id="pills-tab" role="tablist">
```

Comment:
lists are another good opportunity for either a component, or a simple map or filter function over an array to return the li components you need to display